### ISSUE TITLE
Idea: use web workers for services

### ISSUE THREAD

**Issue by nelsonpecora** [nymag/dollar-slice#18]
- controllers talk to the dom
- services (for the most part) don't
- maybe use web workers for services?
- check browser support
- check compilation situation

**Comment by ughitsaaron** [comment]
Maybe for some services but not all. I think there might be some edge cases where calling a web worker for some small task might actually hurt performance more than help.

**Comment by nelsonpecora** [comment]
yep. I think the best use cases are validators (which don't need to talk with the dom) and api calls (the stuff in `edit/` though there are some dom things in there right now)
